.gitignore

```diff
@@ -47,3 +47,8 @@ XmppAddr.hrl
 /_build
 /mnesiadb
 /.rebar
+
+.idea/
+ejabberd.iml
+*log/
+mix.lock
```

Comment:
Hm I don't see us doing this elsewhere - you may have a version mismatch, but I guess we can leave this.